README.md

```diff
@@ -251,7 +255,9 @@ This command creates a random model, which appears at .
 
 ```bash
 export MODEL_NAME=000000-bootstrap
-python3 main.py bootstrap --model-save-path="gs://$BUCKET_NAME/models/$MODEL_NAME"
+python3 main.py bootstrap \
+  --working-dir=estimator_working_dir
```

Comment:
Does this need to exist in advance?  If not, you can mention it's created at runtime

Reply:
done